db/migrate/20231207022742_create_form_submissions.rb

```diff
@@ -0,0 +1,19 @@
+class CreateFormSubmissions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :form_submissions do |t|
+      t.string :form_type, null: false
+      t.uuid :benefits_intake_uuid, null: false
```

Comment:
I need assistance with syntax.
does `null: false` make this non-optional?

Not every form submission goes to benefits intake. 
This confusion indicates my design might suspect 😬

Reply:
I will update this to make it nullable.